src/main.ts

```diff
@@ -1,14 +1,26 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
   app.useGlobalPipes(
     new ValidationPipe({
       whitelist: true,
     }),
   );
+
+  const config = new DocumentBuilder()
+    .setTitle('Recipe App')
+    .setDescription('The recipe app API description')
+    .setVersion('1.0')
+    .addTag('Recipes')
```

Comment:
not sure if you need this tag - this might be responsible for showing this, which does not expose anything:
![image](https://github.com/KubaBaniak/RecipeAppNestjs/assets/100123175/c06c0057-5fc7-4cce-b986-799a13f6285d)